Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -890,8 +896,9 @@ def addVertexFitting(
     outputDirRoot: Optional[Union[Path, str]] = None,
     associatedParticles: str = "particles_input",
     trajectories: Optional[str] = None,
-    trackParameters: str = "trackparameters",
+    trackParameters: str = "fittedTrackParameters",
```

Comment:
if we need to change the default, then this change is not compatible with current clients. We should check all clients!

Reply:
I tried to look for them now and adopt the code. I think the rename makes sense because "per default" there will be a kalman pass before the vertexing